cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala

```diff
@@ -168,19 +173,20 @@ class ScalarFunctionsTest extends DocumentingTest {
     }
     section("size()", "functions-size") {
       p(
-        """To return or filter on the size of a list, use the `size()` function.""".stripMargin)
-      function("size(list)", ("list", "An expression that returns a list."))
+        """`size()` returns the number of elements in a list.""".stripMargin)
+      function("size(list)", "An Integer.", ("list", "An expression that returns a list."))
+      considerations("`size(null)` returns `null`.")
       query(
         """RETURN size(['Alice', 'Bob'])""".stripMargin, ResultAssertions((r) => {
           r.toList should equal(List(Map("size(['Alice', 'Bob'])" -> 2)))
         })) {
         resultTable()
-        p("The number of items in the list is returned by the query.")
+        p("The number of elements in the list is returned.")
       }
     }
-    section("Size of pattern expression", "functions-size-of-pattern-expression") {
+    section("size() applied to pattern expression", "functions-size-of-pattern-expression") {
       p(
-        """This is the same `size()` method described before, but instead of passing in a list directly, you provide a pattern expression that can be used in a match query to provide a new set of results.
+        """This is the same `size()` method as described above, but instead of passing in a list directly, pattern expression can be provided that can be used in a match query to provide a new set of results.
```

Comment:
replace `"...  pattern expression can be provided that can be used in ..."`
with `"...  A pattern expression can be provided that can be used in ..."`?

Reply:
👍